src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,140 +1,121 @@
-   * - **left** yields a tooltip that appears to the left of the trigger
+   * The trigger element the tooltip appears next to.
    */
-  align?: 'right' | 'below' | 'left';
+  children?: React.ReactElement;
   /**
-   * The visually-hidden button text for the tooltip trigger button
+   * The trigger element the tooltip appears next to.
+   *
+   * Use this instead of `children` if the trigger element is being
+   * stored in a ref. Most cases will use `children` and not
+   * `reference`.
    */
-  buttonText?: string;
+  reference?: React.RefObject<Element> | Element;
   /**
-   * Child node(s) that can be nested inside component. This gets displayed inside a `TextPassage` inside the tooltip
+   * The content of the tooltip bubble.
    */
-  children?: ReactNode;
+  content?: React.ReactNode;
   /**
-   * CSS class names that can be appended to the component.
+   * Where the tooltip should be placed in relation to the element it's attached to.
+   *
+   * Tippy also supports 'top-start', 'top-end', 'right-start', 'right-end', etc,
+   * but our CSS currently only supports the 4 main sides.
+   */
+  align?: 'top' | 'right' | 'bottom' | 'left';
+  /**
+   * Whether the tooltip is always visible or always invisible.
+   *
+   * This is most often left undefined so the Tooltip component
+   * controls if/when the bubble appears (on hover, click, focus, etc).
+   */
+  visible?: boolean;
+  /**
+   * Custom classname for additional styles.
+   *
+   * These styles will only affect the tooltip bubble.
    */
   className?: string;
   /**
-   * HTML id for the component
+   * Whether the tooltip has a light or dark background.
    */
-  id?: string;
-}
-
-/**
- * Primary UI component for user interaction
- */
-export const Tooltip: React.FC<Props> = ({
-  className,
-  children,
-  align,
-  buttonText,
-  ...other
-}) => {
+  variant?: 'light' | 'dark';
   /**
-   * Initialize variables and states
+   * How long to delay the Tooltip showing and hiding, in milliseconds.
+   *
+   * If a single number is provided, it will be applied to showing and hiding.
+   * If an array with 2 numbers is provided, the first will apply to showing and
+   * the second will be applied to hiding.
+   * https://atomiks.github.io/tippyjs/v6/all-props/#delay
    */
-  const ref = useRef() as MutableRefObject<HTMLDivElement>;
-  const buttonRef = useRef() as MutableRefObject<HTMLButtonElement>;
-
-  const [isActive, setIsActive] = useState(false);
-
+  delay?: number | [number | null, number | null];
   /**
-   * Toggle isActive state on click
+   * Behavior of the tooltip transition, defaults to an opacity "fade".
+   * Animation guidelines are provided in https://atomiks.github.io/tippyjs/v5/animations/.
+   * A false value will disable animations.
    */
-  function onClick(e: any) {
-    e.preventDefault();
-    setIsActive(!isActive);
-  }
+  animation?: string | boolean;
+};
 
-  useEffect(() => {
-    if (isActive) {
```

Comment:
Interesting that their version of the component is focusable. I wonder why.

Reply:
Hmmm in my understanding, tooltip should [not be focusable](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role#keyboard_interactions)

Reply:
maybe they were trying to handle tooltips w/ interactive content 🤔  we don't want that tho